static/src/javascripts/projects/common/views/acquisitions-epic-design-variations.html

```diff
@@ -0,0 +1,34 @@
+<div class="contributions__epic <%=epicClass%>">
+    <div>
+        <h2 class="contributions__title contributions__title--epic <%=epicTitleClass%>">
+            Since you’re here …
+        </h2>
+        <p class="contributions__paragraph contributions__paragraph--epic <%=p1Class%>">
+            <%=p1%><span class="<%=p1HighlightClass%>"><%=p1Highlight%></span><%=p1PostHighlight%>
+        </p>
+        <p class="contributions__paragraph contributions__paragraph--epic">
+            <%=p2%>
+        </p>
+        <p class="contributions__paragraph contributions__paragraph--epic">
+            <%=p3%>
+        </p>
+    </div>
+
+    <div class="contributions__amount-field">
+        <div>
+            <a class="contributions__option-button contributions__contribute contributions__contribute--epic contributions__contribute--epic-member contributions__contribute--epic-member-top <%=supporterButtonClass%>"
+               href="https://membership.theguardian.com/supporter"
+               target="_blank">
```

Comment:
Please also add `rel="noopener noreferrer"`. I can't stop mentioning the security implications :) https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/